client/src/components/Bills/BillList.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import BillForm from "./BillForm";
+
+const emptyBill = {
+  transaction_date: "",
+  subtotal: null,
+  tax_rate: null,
+  tax_amount: null,
+  tip_rate: null,
+  tip_total: null,
+  who_paid: null,
+  paid_up: null,
+  bill_notes: "",
+  full_total: null,
+};
+
+function BillList() {
+  //This needs an empty array, or the whole thing breaks
+  const [bills, setBills] = useState([]);
+  const [editBillById, setEditBillById] = useState(null);
+
+  useEffect(() => {
+    fetch("http://localhost:5005/db/bills")
+      .then((response) => response.json())
+      .then((billData) => {
+        setBills(billData);
+      });
```

Comment:
💡 production code should have a `catch` here.

Reply:
These were added in PR #3 (should be in main now, but not in this branch).